### ISSUE TITLE
rdki install fails to update packages

### ISSUE THREAD

**Issue by Project0** [oneclick/rubyinstaller2#220]
## What problems are you experiencing?
`rdki install 1 2 3` fails to update the packages as the pacman database is outdated or missing

> warning: database file for 'ucrt64' does not exist (use '-Sy' to download)
> :: Starting core system upgrade...
> there is nothing to do
> :: Starting full system upgrade...
> :: Replace mingw-w64-x86_64-pkg-config with mingw64/mingw-w64-x86_64-pkgconf? [Y/n]
> :: Replace pkg-config with msys/pkgconf? [Y/n] error: failed to prepare transaction (could not find database)

Does someone know a workaround?

## Steps to reproduce
`rdki install 1 2 3`

## What's the output from `ridk version`?
2.6.6-2

**Comment by Project0** [comment]
just fixed it by running those commands before the install
```
  ridk exec bash -c "pacman-key --init"
  ridk exec bash -c "pacman-key --populate"
  ridk exec bash -c "pacman -Syu --noconfirm"
```

**Comment by larskanis** [comment]
@Project0 Did you start `ridk install 1 2 3` a second time? It should succeed when running twice.

Nevertheless this is fixed by commit 9017818c301aa92041a647e16421c3a9d62fe1bc and will be in the next release coming soon.
